### PR TITLE
user form for volunteer

### PR DESCRIPTION
**Pull request by Wolfjawan**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,16 +0,0 @@
-{
```
Comment (kkarimi) [inline .vscode/launch.json]: Don't delete

File: config-overrides.js [hunk 2]
```diff
@@ -1,26 +0,0 @@
-/**
```
Comment (kkarimi) [inline config-overrides.js]: Don't delete

File: cypress.json [hunk 3]
```diff
@@ -1,10 +0,0 @@
-{
```
Comment (kkarimi) [inline cypress.json]: Don't delete

File: config-overrides.js (hunk 2 above)
Comment (Wolfjawan) [reply config-overrides.js]: Why create react app needs a config file, you sid it does create it itself

**Pull request merged**